examples/compliance.mql.yaml

```diff
@@ -19,6 +19,10 @@ policies:
       title: SSH config editing should be limited to admins
       mql: sshd.config.file.permissions.mode == 0644
       impact: 100
+    queries:
```

Comment:
do we need do add query pack dependencies to support this in the maps?

similar to `policy_dependencies` but for query packs instead

Reply:
oh good point! I'm not sure on the wiring we are using internally, but essentially we just want to point people to the MRN of the policy or querypack they want to activate at the end of the day

Reply:
I added a `query_pack_dependencies` field to the proto. We use the policy/pack deps when compiling the framework maps to ensure that the policies/packs are in the bundle.

Note that packs come back as policies in the bundle but we only reference them by `uid` so that should be fine